pkgs/development/interpreters/spidermonkey/78.nix

```diff
@@ -52,7 +51,7 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     autoconf213
     cargo
-    llvmPackages_12.llvm # for llvm-objdump
+    rustc.llvm # for llvm-objdump
```

Comment:
I believe this is built for the host system rather than the build system, so this might cause problems when cross-compiling. See: https://github.com/NixOS/nixpkgs/blob/6b5006b249f007141692a3f294b01b13414c7516/pkgs/development/compilers/rust/1_58.nix#L33-L36 (`llvmShared == rustc.llvm`, `llvmPackagesForBuild == rustc.llvmPackages`)

Reply:
Great catch!  What confusing naming.  Fixed.